docs/api.md

```diff
@@ -96,11 +124,36 @@ Mixins that are bound to several different types can find the current API type i
 or `type` members. [`this.type`](#apitype) would for instance return `GenericObject` and
 [`this.genericType`](#apigenerictype) would return `barchart`.
 
-See the [Mixins examples](/examples/README.md#mixins) on how to use it.
+See the [Mixins examples](/examples/README.md#mixins) on how to use it, below is an outline of what the mixin
+API consists of.
+
+### `mixin.init(args)`
+
+This function will be executed for each generated API.
+
+See below what `args` contains.
+
+| Property                | Type     | Description |
+|-------------------------|----------|-------------|
+| `config`                | Object   | A reference to the normalized enigma.js configuration object. |
```

Comment:
I might reformulate this, I just want to clarify that it's a configuration containing e.g. `Promise` and others that we default to if they are not passed in.